api/resources_portal/views/document_views.py

```diff
@@ -0,0 +1,330 @@
+from django.utils.decorators import method_decorator
+from rest_framework import serializers
+
+from django_elasticsearch_dsl_drf.constants import LOOKUP_FILTER_RANGE, LOOKUP_QUERY_IN
+from django_elasticsearch_dsl_drf.filter_backends import (
+    CompoundSearchFilterBackend,
+    DefaultOrderingFilterBackend,
+    FacetedSearchFilterBackend,
+    FilteringFilterBackend,
+    OrderingFilterBackend,
+)
+from django_elasticsearch_dsl_drf.pagination import LimitOffsetPagination as ESLimitOffsetPagination
+from django_elasticsearch_dsl_drf.viewsets import DocumentViewSet
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+from elasticsearch_dsl import TermsFacet
+from six import iteritems
+
+from resources_portal.models import Material, Organization, User
+from resources_portal.models.documents import MaterialDocument, OrganizationDocument, UserDocument
+
+
+# Puts material search data into a form that the client will accept
+class MaterialDocumentSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    category = serializers.CharField(read_only=True)
+    title = serializers.CharField(read_only=True)
+    url = serializers.CharField(read_only=True)
+    pubmed_id = serializers.CharField(read_only=True)
+    additional_metadata = serializers.CharField(read_only=True)
+    created_at = serializers.DateField(read_only=True)
+    updated_at = serializers.DateField(read_only=True)
+    organism = serializers.CharField(read_only=True)
+    has_publication = serializers.CharField(read_only=True)
+    has_pre_print = serializers.CharField(read_only=True)
+
+    class Meta:
+        model = Material
+        fields = (
+            "id",
+            "category",
+            "title",
+            "url",
+            "pubmed_id",
+            "additional_metadata",
+            "created_at",
+            "updated_at",
+            "organism",
+            "has_publication",
+            "has_pre_print",
+        )
+        read_only_fields = ("id", "created_at", "updated_at")
```

Comment:
This whole route is read-only right?

Reply:
Yes. Removed the reference to this here, as it's handled in the view.